### DISCUSSION TITLE
How to Send a Conversation Message using "conversation_id"

### DISCUSSION THREAD

**Discussion by liberalchang** [langgenius/dify#9496] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

** How do I use the "conversation_id" from the last post to continue the conversation during the conversation **
This is my err：dify_client.errors.DifyResourceNotFound: status_code=404, code=not_found, Conversation Not Exists.
![image](https://github.com/user-attachments/assets/2a99a606-125c-46bc-8699-7bdf1d0ce2b8)
![image](https://github.com/user-attachments/assets/c034aaf5-4c50-4bba-b7e7-29db624ab998)

**Comment by bensonlv** [comment]
I also have this issue. what i want to let user access a login webpage, then pass the login token back to the conversation which talk to the chatbot flow that this use is validated, they can answer the confidential message. but when i tried call the chat-message with a conversation_id, it always show the 'conversation not exits.'
Any one can help for this?
